.circleci/config.yml

```diff
@@ -138,52 +37,12 @@ workflows:
           context: genesis-svn
           requires:
             - approval-for-deploy-tested-up-to-bump
-
-  branch_deploy:
-    jobs:
-      - checkout:
-          filters:
-            branches:
-              only:
-                - master
-      - checks:
-          requires:
-            - checkout
-          filters:
-            branches:
-              only:
-                - master
-      - deploy_svn_branch:
+      - wp-svn/deploy:
```

Comment:
This is the main fix. 

It uses [our Orb](https://circleci.com/developer/orbs/orb/studiopress/wp-svn), which handles almost all of the `svn` details.